_posts/2019-12-13-cockpit-209.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Cockpit 209
+author: mvo
+date: 2019-12-13
+tags: cockpit linux
+slug: cockpit-209
+category: release
+summary: Cockpit with Overview improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 209.
+
+### A new design for the Overview page
+
+The landing page has been completely redesigned.  Information is
+grouped into easier to understand panels, health information is much
+more prominent, the resource graphs have been moved to their own page,
+and the hardware information page should now be easier to find.
+
+![Overview](/images/new-overview.png)
+
+### Session timeouts
+
+Cockpit will automatically log you out of your current session after
+15 minutes of inactivity.
+
+![Automatic logout](/images/session-timeout.png)
+
+You can configure the timeout in `/etc/cockpit/cockpit.conf`.  For
+example, this will disable automatic logouts completely:
+```
+[Session]
+IdleTimeout=0
+```
+
+### Support for banners on the login page
+
+Like SSH, Cockpit can now optionally show the content of banner files
+on the login screen.
+
+![/etc/issue](/images/etc-issue.png)
+
+This need to be configured in `/etc/cockpit/cockpit.conf`.  For example:
+```
+[Session]
+Banner=/etc/issue.cockpit
+```
+
+### Support for TLS client certificate authentication
+
+You can now log into Cockpit with a TLS client certificate that is
+provided by your browser or a device like a Smart Card or YubiKey.
+
+See it in action:
+
+<iframe width="960" height="540" src="https://www.youtube.com/embed/v07cHZQsRyo" frameborder="0" allowfullscreen></iframe>
+
+And read the documentation [here](https://cockpit-project.org/guide/latest/cert-authentication.html).
+
+### Support for Fedora CoreOS
+
+[Fedora CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/) is a minimal and robust, Fedora based operating system for hosting container workloads. It is the successor of [Fedora Atomic Host](http://www.projectatomic.io/), which is not supported any more.
+
+We now fully support running Cockpit on this platform.
+
+Please see the [Running Cockpit](https://cockpit-project.org/running.html) documentation or the [cockpit/ws docker hub](https://hub.docker.com/r/cockpit/ws/) page for how to install Cockpit on Fedora CoreOS.
+
+### Dropped support for the pam_rhost module
+
+The `pam_rhosts` module is not in Cockpit's PAM stack by default (not
+in any other stack) and the remote IP is not a reliable piece of
+information.  NAT, VPNs, or reverse proxies (such as Cockpit's own
+cockpit-tls) all destroy the actual IP of the browser's machine.
+
```

Comment:
We should have a sentence here (other than the heading) which states the change explicitly.

Reply:
Yes, added.